insitu/version.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define ISAAC_VERSION_MAJOR 0
+#define ISAAC_VERSION_MINOR 1
+#define ISAAC_VERSION_PATCH 0
+#define ISAAC_VERSION_TWEAK 0
```

Comment:
do we really need tweaks? ;) just makes everything longer, use more patch levels :)